llama_cpp/server/app.py

```diff
@@ -504,14 +495,9 @@ async def create_chat_completion(
     if body.grammar is not None:
         kwargs["grammar"] = llama_cpp.LlamaGrammar.from_string(body.grammar)
 
-    if body.min_tokens > 0:
-        _min_tokens_logits_processor = llama_cpp.LogitsProcessorList(
-            [llama_cpp.MinTokensLogitsProcessor(body.min_tokens, llama.token_eos())]
-        )
-        if "logits_processor" not in kwargs:
-            kwargs["logits_processor"] = _min_tokens_logits_processor
-        else:
-            kwargs["logits_processor"].extend(_min_tokens_logits_processor)
+    # Override max_tokens with max_completion_tokens.
+    if body.max_completion_tokens is not None:
+        kwargs["max_tokens"] = body.max_completion_tokens
```

Comment:
It's best to use `max_completion_tokens` in `kwargs` and throughout the whole codebase to keep the terminology consistent, if it's not too complicated to change right now.

Reply:
`max_tokens` is used pervasively throughout the code. While I could globally search/replace it, I want to limit our edits to `llama_cpp/server/*` if possible. I still believe, that we will be able to include python bindings as a dependency (i.e. use llama-cpp-python as a library directly from upstream) and only have "Server" part custom. I don't think we are ready to maintain bindings to llama.cpp ourselves.

Reply:
In that case, yes, let's keep it as is!